nsxt/resource_nsxt_edge_transport_node.go

```diff
@@ -29,47 +31,56 @@ var ipAssignmentTypes = []string{
 	"static_ip_pool",
 }
 
+var mpTypeFromPolicyType = map[string]string{
```

Comment:
If the code assumes MP and Policy values are the same, do we need this translation?

Reply:
We query the host switch profile type using this policy [API](https://dp-downloads.broadcom.com/api-content/apis/API_NTDCRA_001/4.2.1/html/api_includes/method_GetPolicyHostSwitchProfile.html) which returns policy types, e.g `PolicyUplinkHostSwitchProfile`.
The switch type consumes [HostSwitchSpec](https://dp-downloads.broadcom.com/api-content/apis/API_NTDCRA_001/4.2.1/html/api_includes/types_StandardHostSwitch.html) type has `host_switch_profile_id` attribute which is a string - which is enumerated by [HostSwitchProfileType](https://dp-downloads.broadcom.com/api-content/apis/API_NTDCRA_001/4.2.1/html/api_includes/types_HostSwitchProfileType.html) that has no "Policy" prefix to its values. Hence translation is required even though it's not MP.
However `mpTypeFromPolicyType` name is bad - it's just `HostSwitchProfileType` translation and not any Policy->MP translation. I'll rename this var.